### PR TITLE
Header

### PR DESCRIPTION
**Pull request by SilviaBarcellini**
add header component, which will appear on the top section of every page of the app.
it contains the following components: 
- logo
- donut icon

### CODE REVIEW

File: components/card/card.tsx [hunk 1]
```diff
@@ -0,0 +1,19 @@
+import styles from "./card.module.css";
+
+export type CardProps = {
+  title: string;
+  image: string;
+};
+
+function Card({ title, image, ...props }: CardProps) {
```
Comment (phlgr) [inline components/card/card.tsx:8]: You could remove `...props` here as well

File: components/footer/favs/favsBtn.stories.tsx [hunk 2]
```diff
@@ -0,0 +1,16 @@
+import React from "react";
```
Comment (phlgr) [inline components/footer/favs/favsBtn.stories.tsx:1]: You don't need to import this here. Remove this before merging.

File: components/footer/favs/favsBtn.tsx [hunk 3]
```diff
@@ -0,0 +1,15 @@
+import styles from "./favsBtn.module.css";
+/* import FavsImg from "./favsBtn.stories"; */
+
+function FavsBtn() {
+  return (
+    <a href="http://localhost:3000/">
```
Comment (phlgr) [inline components/footer/favs/favsBtn.tsx:6]:
```suggestion
    <a href="#">
```
Remove the link to the localhost as a placeholder

File: components/footer/home/homeBtn.stories.tsx [hunk 4]
```diff
@@ -0,0 +1,17 @@
+import React from "react";
```
Comment (phlgr) [inline components/footer/home/homeBtn.stories.tsx:1]: Remove this line as well

File: components/footer/home/homeBtn.tsx [hunk 5]
```diff
@@ -0,0 +1,22 @@
...
+
+/* import homeImg from "./homeBtn.stories"; */
+
+export type HomeBtnProps = {
+  src: string;
+};
+
+function HomeBtn({ ...props }: HomeBtnProps) {
+  return (
+    <a href="http://localhost:3000/">
```
Comment (phlgr) [inline components/footer/home/homeBtn.tsx:11]: Same here
```suggestion
    <a href="#">
```

File: components/greetings/greetings.tsx [hunk 6]
```diff
@@ -0,0 +1,23 @@
...
+
+if (currentTime < 12) {
+  greetings = "Good Morning";
+} else if (currentTime < 18) {
+  greetings = "Good Afternoon";
+} else if (currentTime < 22) {
+  greetings = "Good Evening";
+} else {
+  greetings = "Good Night";
+}
```
Comment (phlgr) [inline components/greetings/greetings.tsx:17]: Move this into the function and into a `useEffect` to only calls this once the page has loaded

File: components/header/header.tsx [hunk 7]
```diff
@@ -0,0 +1,19 @@
+import styles from "./header.module.css";
+
+export type HeaderProps = {
+  logo: string;
+  imageSrc: string;
+};
+
+function Header({ logo, imageSrc, ...props }: HeaderProps) {
```
Comment (phlgr) [inline components/header/header.tsx:8]: `...props` here as well

File: components/logo/logo.tsx [hunk 8]
```diff
@@ -0,0 +1,18 @@
+import styles from "./logo.module.css";
+
+export type LogoProps = {
+  logo: string;
+  size: string;
+};
+
+function Logo({ logo, size, ...props }: LogoProps) {
```
Comment (phlgr) [inline components/logo/logo.tsx:8]: `...props` here as well

**Review by phlgr — CHANGES_REQUESTED** [review]
A deployment here would be really helpful
Nonetheless, GJ 🚀

File: components/btn/btn.tsx [hunk 9]
```diff
@@ -0,0 +1,15 @@
+import styles from "./btn.module.css";
+
+export type BtnProps = {
+  label: string;
+};
+
+function Btn({ label, ...props }: BtnProps) {
```
[the remark this answers is not in the document]
Comment (SilviaBarcellini) [reply components/btn/btn.tsx:7]: Will do, thanks!

[the remark this answers is not in the document]
Comment (SilviaBarcellini) [reply components/btn/btn.tsx:7]: Will do, thanks!

File: components/greetings/greetings.tsx (hunk 6 above)
Comment (SilviaBarcellini) [reply components/greetings/greetings.tsx:17]: Will open a further Pull Request for it, thanks!

**Pull request merged**